.github/composite/getLatest/action.yml

```diff
@@ -0,0 +1,17 @@
+name: Get latest release
```

Comment:
I would rename the folder in which this file lives (and thus the name of the action) to `download`.

Reply:
I'll resolve this comment since it is already covered by my comment on the README file where I propose a different folder herarchy.